README.md

```diff
@@ -3,7 +3,7 @@
 This repository contains a boilerplate static site generator setup for creating CNCF documentation projects. We strongly recommend using this setup (it helps us help you and your project!), but none of the technologies in the stack are strictly required.
 
 The starter uses the following:
-* **[Hugo](https://gohugo.io/)** as a static site generator
+* **[Hugo(This requires Hugo 9.73+ extended version)](https://gohugo.io/)** as a static site generator
```

Comment:
Some notes:
* Could you add a space between `Hugo` and the `(`.
* I'm also not sure we need the "This requires Hugo" part, just the version and extended modifier should be enough.
* I also think the original issue had a typo in the version requirement. I think probably we need Hugo's extended v0.73 or above, not version 9.73.
```suggestion
* **[Hugo (extended, v0.73 or above)](https://gohugo.io/)** as a static site generator
```